src/UI/MapView.java

```diff
@@ -34,10 +34,8 @@ public InGameMenu<? extends Object> getCurrentGameMenu()
     return mapController.getCurrentGameMenu();
   }
 
-  /**
-   * Adds the new events to the queue so they can be animated.
-   */
-  public abstract void animate( GameEventQueue newEvents );
+  public abstract boolean shouldAnimate( GameEvent toAnimate );
+  public abstract void animate( GameEvent toAnimate );
```

Comment:
I suspect you have uncommitted changes - InfoView is whining that it needs to inherit the redefined `animate(GameEvent)` from `MapView`.

Reply:
No, actually, somehow. It complains there's an error when you try to run it, but the `InfoView` itself works.
I guess I didn't notice because I have a bunch of untracked files in there that Eclipse whines are errors.